src/Tizen.NUI/src/internal/WebView/WebContext.cs

```diff
@@ -33,9 +33,9 @@ public class WebContext : Disposable
         private float timeOffset;
         private ApplicationType applicationType;
         private WebSecurityOriginList securityOriginList;
-        private SecurityOriginListAcquiredCallback securityOriginsAcquiredCallback;
+        private SecurityOriginListAcquiredCallback securityOriginListAcquiredCallback;
         private readonly WebContextSecurityOriginListAcquiredProxyCallback securityOriginListAcquiredProxyCallback;
-        private WebPasswordDataList passwordList;
+        private WebPasswordDataList passwordDataList;
```

Comment:
Is passwordData related with Interceptor API?

Reply:
I think the patch is including code that are not related with request interceptor.

Reply:
Done.